docker-compose/docker-compose.yml

```diff
@@ -13,8 +13,8 @@ services:
       - $NATS_SURVEYOR_CREDS:/etc/surveyor/SYS.creds
       - ./observations:/observations
       - ./jetstream:/jetstream
-    entrypoint: /nats-surveyor
-    command: -c ${NATS_SURVEYOR_SERVER_COUNT} -creds /etc/surveyor/SYS.creds -s "${NATS_SURVEYOR_SERVERS}" -observe /observations -jetstream /jetstream
+    entrypoint: /entrypoint.sh
```

Comment:
Can you remove the entrypoint so it will use the image default?

Reply:
Done..